File: datasets/IR-Plag/convertai-original-java-rest-python/case-05/plagiarized/L3/05/method.py
import java.util.Scanner

def beautyReverse(num):
    while num != 0:
        r = num % 10
        print(r, end='')
        num = num // 10
    print()

def main():
    print("Enter an integer: ", end='')
    n = int(input())
    beautyReverse(n)

if __name__ == "__main__":
    main()

File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-05/plagiarized/L2/04/method.kt
package IR

import java.util.Scanner

fun main() {
    print("Enter an integer: ")
    val input = Scanner(System.`in`)
    val n = input.nextInt()
    beautyReverse(n)
}

fun beautyReverse(num: Int) {
    var tempNum = num
    while (tempNum != 0) {
        val r = tempNum % 10
        print(r)
        tempNum /= 10
    }
    println()
}

